setup.py

```diff
@@ -114,7 +114,7 @@ def run(self):
             cmd & FG
 
             cmd = (local['conan']['create']['.']
-                ['--version']['0.6.5']
+                ['--version'][get_version()]
```

Comment:
Beautiful, Pablo. I have to add this for `libqasm` too. Many thanks for this fix!